test/range/count.wren

```diff
@@ -0,0 +1,25 @@
+// Ordered range.
+IO.print((2..5).count) // expect: 4
+IO.print((3..3).count) // expect: 1
+IO.print((0..3).count) // expect: 4
+IO.print((-5..3).count) // expect: 9
+IO.print((-5..-2).count) // expect: 4
+
+// Backwards range.
+IO.print((5..2).count) // expect: 4
+IO.print((3..0).count) // expect: 4
+IO.print((3..-5).count) // expect: 9
+IO.print((-2..-5).count) // expect: 4
+
+// Exclusive ordered range.
+IO.print((2...5).count) // expect: 3
+IO.print((3...3).count) // expect: 0
+IO.print((0...3).count) // expect: 3
+IO.print((-5...3).count) // expect: 8
+IO.print((-5...-2).count) // expect: 3
+
+// Exclusive backwards range.
+IO.print((5...2).count) // expect: 3
+IO.print((3...0).count) // expect: 3
+IO.print((3...-5).count) // expect: 8
+IO.print((-2...-5).count) // expect: 3
```

Comment:
Arg, you're right, double can't do this because it is not precise when fractions are involved.
```lua
print(6.1 - 3.1, math.floor(6.1 - 3.1), 3.0, math.floor(3.0))
-- output: 3.0	2	3.0	3
```
Now I wonder if implementing this correctly with fractions is possible at all. :-/